cmd/kots/cli/velero.go

```diff
@@ -352,6 +353,19 @@ func VeleroConfigureOtherS3Cmd() *cobra.Command {
 				return errors.New("velero namespace not found")
 			}
 
+			var caCertData []byte
+			caCertFile := v.GetString("cacert")
+			if caCertFile != "" {
+				realPath, err := filepath.Abs(caCertFile)
```

Comment:
why is this needed?

Reply:
TBH I got this directly from the Velero CLI. My assumption is that it's because `Abs` also runs [Clean](https://golang.org/pkg/path/filepath/#Clean) which can handle some corner case inputs. I tested and it works without, but I'm inclined to keep unless there are strong objections.

Reply:
sounds good